Add PT-BR automatic translation maintenance_github_grooming.pt.Rmd (#816)
ropensci/dev_guide: maintenance_github_grooming.pt.Rmd

> ---

Author: ```suggestion

```

> aliases: grooming.html

Author: ```suggestion

```

> ---

Author: ```suggestion

```

> # GitHub Grooming {#grooming}

Reviewer: ```suggestion
# Preparação do GitHub {#grooming}
```

```{block, type="summaryblock"}

> Atualmente, os pacotes do rOpenSci são, em sua grande maioria, desenvolvidos no GitHub. Aqui estão algumas dicas para aproveitar a plataforma em uma seção sobre [tornar seu repositório mais detectável] (#repodiscoverability) e uma seção sobre [comercializar sua própria conta do GitHub após passar pela revisão por pares] (#marketown).

Reviewer: ```suggestion
Atualmente, os pacotes do rOpenSci são, em sua grande maioria, desenvolvidos no GitHub. Aqui estão algumas dicas para aproveitar a plataforma em uma seção sobre [tornar seu repositório mais detectável](#repodiscoverability) e uma seção sobre [comercializar sua própria conta do GitHub após passar pela revisão por pares](#marketown).
```

```

## Torne seu repositório mais detectável {#repodiscoverability}

### Tópicos do repositório do GitHub {#git-hub-repo-topics}

> GitHub [tópicos do repositório](https://blog.github.com/2017-01-31-introducing-topics/) ajudam a navegar e pesquisar repositórios do GitHub, são usados por [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave do registro rOpenSci.

Reviewer: ```suggestion
[Tópicos de repositórios](https://blog.github.com/2017-01-31-introducing-topics/) do GitHub ajudam a navegar e pesquisar repositórios do GitHub, são usados pelo [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave de registro da rOpenSci.
```

Reviewer 2: ```suggestion
Os [tópicos de repositórios](https://blog.github.com/2017-01-31-introducing-topics/) do GitHub ajudam a navegar e pesquisar repositórios do GitHub, são usados pelo [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são processados pelo [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave de registro da rOpenSci.
```

> Recomendamos que você:

Reviewer: "Recomendamos que você" followed by "adicionar" in the next lines doesn't add up.

```suggestion
Recomendamos:
```

> - Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacotes.

Reviewer 2: ```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao repositório de seu pacote.
```

> - Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacote.

Author: ```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao repositório de seu pacote.
```

> - Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacotes.
> 
> - Adicionar quaisquer outros tópicos relevantes ao repositório de pacotes que você possui.

Reviewer: I assume it is 1 repository per package. Or am I wrong and it is common in R environment to have more than one package per repo?

```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacote.

- Adicionar quaisquer outros tópicos relevantes ao repositório de pacote que você possui.
```

Author: It is correct, it is one repo per package.

> - Adicionar quaisquer outros tópicos relevantes ao repositório de pacotes que você possui.

Reviewer 2: ```suggestion
- Adicionar quaisquer outros tópicos relevantes ao repositório do seu pacote.
```

> - Adicionar quaisquer outros tópicos relevantes ao repositório de pacote que você possui.

Author: ```suggestion
- Adicionar quaisquer outros tópicos relevantes ao repositório do seu pacote.
```

Poderemos fazer sugestões a você depois que seu pacote for integrado.

### GitHub linguist {#git-hub-linguist}

> [GitHub linguist](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".

Author: ```suggestion
O [GitHub linguist](https://github.com/github/linguist) atribuirá uma linguagem ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código em C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade de adicionar os tópicos "r", "r-package" e "rstats".
```

> [Linguista do GitHub](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".

Reviewer 2: ``` suggestion
O [GitHub linguist](https://github.com/github/linguist) atribuirá uma linguagem ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código em C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade de adicionar os tópicos "r", "r-package" e "rstats".
```

> ### Linguista do GitHub {#git-hub-linguist}
> 
> [Linguista do GitHub](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".
> 
> Recomendamos que você substitua o linguista do GitHub adicionando ou modificando um .gitattributes ao seu repositório em dois casos:

Reviewer: ```suggestion
### GitHub linguist {#git-hub-linguist}

[GitHub linguist](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".

Recomendamos que você substitua o GitHub linguist adicionando ou modificando um .gitattributes ao seu repositório em dois casos:
```

> - Se você armazenar arquivos html em locais não padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicionar `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))

Reviewer: ```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

> - Se você armazenar arquivos html em locais não padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicionar `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))

Reviewer 2: ```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo em uso real](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

> - Se você armazenar arquivos html em locais não padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicionar `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))

Reviewer 2: Acho que fica estranho deixar *Exemplo na natureza*. Eu sugeri *Exemplo em uso real*, mas ainda estou na dúvida se é a melhor opção.

> - Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))

Author: ```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo em uso real](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

> - Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo na natureza](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))

Reviewer 2: ```suggestion
- Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo em uso real](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))
```

> Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguista não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).

Reviewer 2: ```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o GitHub linguist não reconhecer corretamente que seu repositório contém principalmente código R, seu pacote não aparecerá nos resultados de pesquisa usando o filtro `language:R`. Da mesma forma, seu repositório não poderá ser listado entre os [repositórios R em alta](https://github.com/trending/r).
```

> Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguist não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).

Author: ```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o GitHub linguist não reconhecer corretamente que seu repositório contém principalmente código R, seu pacote não aparecerá nos resultados de pesquisa usando o filtro `language:R`. Da mesma forma, seu repositório não poderá ser listado entre os [repositórios R em alta](https://github.com/trending/r).
```

> Mais informações sobre substituições de GitHub linguist [aqui](https://github.com/github/linguist#overrides).

Author: ```suggestion
Mais informações sobre as substituições do GitHub linguist podem ser encontradas [aqui](https://github.com/github/linguist#overrides).
```

> Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguista não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).
> 
> Mais informações sobre substituições de linguistas no GitHub [aqui](https://github.com/github/linguist#overrides).

Reviewer: ```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguist não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).

Mais informações sobre substituições de GitHub linguist [aqui](https://github.com/github/linguist#overrides).
```

> Mais informações sobre substituições de linguistas no GitHub [aqui](https://github.com/github/linguist#overrides).

Reviewer 2: ```suggestion
Mais informações sobre as substituições do GitHub linguist podem ser encontradas [aqui](https://github.com/github/linguist#overrides).
```

## Comercialize sua própria conta {#marketown}

> - Como autor de um pacote integrado, você agora é membro da organização "ropensci" do rOpenSci no GitHub. Por padrão, as associações à organização são privadas; consulte [como torná-la pública nos documentos do GitHub](https://help.github.com/articles/publicizing-or-hiding-organization-membership/).

Reviewer: ```suggestion
- Como autor de um pacote integrado, você agora é membro da organização "ropensci" da rOpenSci no GitHub. Por padrão, as participações da organização são privadas; consulte [como torná-la pública na documentação do GitHub](https://help.github.com/articles/publicizing-or-hiding-organization-membership/).
```

> - Mesmo depois que o repositório do seu pacote tiver sido transferido para o rOpenSci, você pode [fixá-lo em sua própria conta](https://help.github.com/articles/pinning-repositories-to-your-profile/).

Reviewer 2: ```suggestion
- Mesmo após o repositório do seu pacote ser transferido para a rOpenSci, você pode [fixá-lo em sua conta pessoal](https://help.github.com/articles/pinning-repositories-to-your-profile/).
```
